pkg/apis/policy/v1alpha1/types_certificaterequestpolicy.go

```diff
@@ -172,118 +174,123 @@ type CertificateRequestPolicyAllowedX509Subject struct {
 	// +optional
 	PostalCodes *CertificateRequestPolicyAllowedStringSlice `json:"postalCodes,omitempty"`
 
-	// SerialNumber defines the X.509 Subject Serial Number that may be requested
-	// for.
+	// SerialNumber defines the X.509 Subject Serial Number that may be
+	// requested for.
```

Comment:
```suggestion
	// SerialNumber defines the X.509 Subject Serial Number that may be
	// requested.
```

This is weird. What kind of format is this? If it's a single value, that makes no sense...

Reply:
It is equivalent to the `CommonName` field. While the `CommonName` field might be used, I also think this field makes little sense. But we cannot remove it without breaking the API.

Reply:
Slightly reworded. Not sure if I can improve more.